### PR TITLE
VXL ebuild

### PR DESCRIPTION
**Pull request by TheChymera**
Package for VXL in an effort to move #401 forward.

Currently fails with [this log here](https://bpaste.net/show/b2704cb9636a).

the latter part of which reads:

```
[ 21%] Building CXX object contrib/brl/bseg/bstm/basic/CMakeFiles/bstm_basic.dir/bstm_block_id.o
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/bstm/basic && /usr/bin/x86_64-pc-linux-gnu-g++  -DVXL_LEGACY_ERROR_REPORTING -DVXL_WARN_DEPRECATED -DVXL_WARN_DEPRECATED_ONCE  -DNDEBUG -march=native -O2 -pipe  -fPIC -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/vcl -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/vcl -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/core -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/core -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bbas -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bbas -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bseg -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bpro -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/b3p/expatpp    -o CMakeFiles/bstm_basic.dir/bstm_block_id.o -c /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bseg/bstm/basic/bstm_block_id.cxx
make[2]: Leaving directory '/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build'
make -f contrib/brl/bseg/boxm/tests/CMakeFiles/boxm_tests.dir/build.make contrib/brl/bseg/boxm/tests/CMakeFiles/boxm_tests.dir/build
make[2]: Entering directory '/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build'
/usr/bin/cmake -E cmake_progress_report /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/CMakeFiles 14
[ 22%] Building CXX object contrib/brl/bseg/boxm/tests/CMakeFiles/boxm_tests.dir/test_utils.o
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/boxm/tests && /usr/bin/x86_64-pc-linux-gnu-g++  -DVXL_LEGACY_ERROR_REPORTING -DVXL_WARN_DEPRECATED -DVXL_WARN_DEPRECATED_ONCE  -DNDEBUG -march=native -O2 -pipe  -fPIC -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/vcl -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/vcl -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/core -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/core -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bbas -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bbas -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bseg -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bpro -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/gel -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/gel/mrc -I/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/b3p/expatpp    -o CMakeFiles/boxm_tests.dir/test_utils.o -c /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999/contrib/brl/bseg/boxm/tests/test_utils.cxx
Linking CXX executable negate
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/sbin && /usr/bin/cmake -E cmake_link_script CMakeFiles/negate.dir/link.txt --verbose=1
Linking CXX executable vil_convert
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/sbin && /usr/bin/cmake -E cmake_link_script CMakeFiles/vil_convert.dir/link.txt --verbose=1
/usr/bin/x86_64-pc-linux-gnu-g++   -march=native -O2 -pipe  -fPIC   -Wl,-O1 -Wl,--as-needed  CMakeFiles/negate.dir/negate.o  -o negate -rdynamic ../../../../lib/libvul.a ../../../../lib/libvil.a ../../../../lib/libvcl.a -lm -ljpeg ../../../../lib/libgeotiff.a -ltiff -lpng -lz ../../../../lib/libopenjpeg2.a 
/usr/bin/x86_64-pc-linux-gnu-g++   -march=native -O2 -pipe  -fPIC   -Wl,-O1 -Wl,--as-needed  CMakeFiles/vil_convert.dir/vil_convert.o  -o vil_convert -rdynamic ../../../../lib/libvul.a ../../../../lib/libvil.a ../../../../lib/libvcl.a -lm -ljpeg ../../../../lib/libgeotiff.a -ltiff -lpng -lz ../../../../lib/libopenjpeg2.a 
Linking CXX static library ../../../../../lib/libbstm_basic.a
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/bstm/basic && /usr/bin/cmake -P CMakeFiles/bstm_basic.dir/cmake_clean_target.cmake
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/bstm/basic && /usr/bin/cmake -E cmake_link_script CMakeFiles/bstm_basic.dir/link.txt --verbose=1
/usr/bin/x86_64-pc-linux-gnu-ar cq ../../../../../lib/libbstm_basic.a  CMakeFiles/bstm_basic.dir/bstm_block_id.o
/usr/bin/x86_64-pc-linux-gnu-ranlib ../../../../../lib/libbstm_basic.a
make[2]: Leaving directory '/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build'
/usr/bin/cmake -E cmake_progress_report /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/CMakeFiles 
[ 22%] Built target bstm_basic
make[2]: Leaving directory '/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build'
/usr/bin/cmake -E cmake_progress_report /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/CMakeFiles 
[ 22%] Built target vil_convert
make[2]: Leaving directory '/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build'
/usr/bin/cmake -E cmake_progress_report /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/CMakeFiles 
[ 22%] Built target negate
Linking CXX static library ../../../../../lib/libboxm_tests.a
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/boxm/tests && /usr/bin/cmake -P CMakeFiles/boxm_tests.dir/cmake_clean_target.cmake
cd /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/contrib/brl/bseg/boxm/tests && /usr/bin/cmake -E cmake_link_script CMakeFiles/boxm_tests.dir/link.txt --verbose=1
/usr/bin/x86_64-pc-linux-gnu-ar cq ../../../../../lib/libboxm_tests.a  CMakeFiles/boxm_tests.dir/test_utils.o
/usr/bin/x86_64-pc-linux-gnu-ranlib ../../../../../lib/libboxm_tests.a
make[2]: Leaving directory '/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build'
/usr/bin/cmake -E cmake_progress_report /var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build/CMakeFiles  14
[ 22%] Built target boxm_tests
make[1]: Leaving directory '/var/tmp/portage/sci-libs/vxl-9999/work/vxl-9999_build'
Makefile:126: recipe for target 'all' failed
make: *** [all] Error 2
 * ERROR: sci-libs/vxl-9999::neurogentoo failed (compile phase):
 *   emake failed
```

### CODE REVIEW

File: licenses/VXL [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Copyright 2000-2013 VXL Contributors
```
Comment (jlec) [inline licenses/VXL]: That license looks like it is already present in the tree. Please use on of those instead.

File: sci-libs/vxl/vxl-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+EGIT_REPO_URI="http://git.code.sf.net/p/vxl/git"
+
+LICENSE="VXL"
+SLOT="0"
+IUSE=""
+KEYWORDS=""
+
+RDEPEND=""
+
+DEPEND=""
```
Comment (jlec) [inline sci-libs/vxl/vxl-9999.ebuild]: Drop empty *DEPEND

File: licenses/VXL (hunk 1 above)
Comment (TheChymera) [inline licenses/VXL]: The only vxl license I see is the one I added myself :-/ amy I missing anything?

```
chymera@zenbookhost ~ $ ls /usr/portage/licenses/V*
/usr/portage/licenses/ValveSDK    /usr/portage/licenses/Vivaldi
/usr/portage/licenses/VGBA        /usr/portage/licenses/Volano
/usr/portage/licenses/Vidyo-EULA  /usr/portage/licenses/VOSTROM
/usr/portage/licenses/VIGRA       /usr/portage/licenses/VTK
chymera@zenbookhost ~ $ ls /usr/portage/licenses/v*
/usr/portage/licenses/ventrilo    /usr/portage/licenses/vlgothic
/usr/portage/licenses/vienna-rna  /usr/portage/licenses/vmd
/usr/portage/licenses/vim         /usr/portage/licenses/vmware
/usr/portage/licenses/vim.org
chymera@zenbookhost ~ $ ls /home/chymera/src/neurogentoo/licenses/V*
/home/chymera/src/neurogentoo/licenses/VXL
chymera@zenbookhost ~ $ ls /home/chymera/src/neurogentoo/licenses/v*
/home/chymera/src/neurogentoo/licenses/vampir
```

**Comment by TheChymera** [comment]
@jlec is my build error expat-related? I see there's quite a few repeats of this line in the (full, patebined) error log:

```
-- Could NOT find EXPATPP (missing:  EXPATPP_LIBRARY EXPATPP_INCLUDE_DIR) 
```

File: licenses/VXL (hunk 1 above)
Comment (jlec) [inline licenses/VXL]: How about this one:

http://opensource.org/licenses/BSD-3-Clause

Comment (TheChymera) [inline licenses/VXL]: Ok, yes, I didn't know the BSD license so I couldn't recognise it---

**Comment by TheChymera** [comment]
@jlec could you help me get rid of all the afni stuff that got into this branch by accident? :(

**Comment by jlec** [comment]
you need to revert commit d055f9d57336587aafb2f7857eebfe7f96360db1

**Comment by TheChymera** [comment]
Jesus christ, I did **EXACTLY** what [this well-reputed answer](http://stackoverflow.com/a/4114122/1893275) instructed:

```
git revert -m 1 d055f9d
git mergetool
git commit
```

Why does git aways have to blow up after it blows up?

@jlec could you maybe help me out here? :(

**Pull request closed without merging**